src/base/emu-i386/kvm.c

```diff
@@ -593,6 +593,9 @@ void mmap_kvm(int cap, unsigned phys_addr, size_t mapsize, void *addr, dosaddr_t
   /* with KVM we need to manually remove/shrink existing mappings */
   do_munmap_kvm(phys_addr, mapsize);
   mmap_kvm_no_overlap(phys_addr, addr, mapsize, 0);
+  /* monitor dirty pages on regular low ram for JIT */
+  if (phys_addr == 0 && IS_EMU() && !CONFIG_CPUSIM)
```

Comment:
Maybe MAPPING_INIT_LOWRAM or
alike rather than the address check?

Reply:
will do

Reply:
done